### DISCUSSION TITLE
Message Bus service unavailable

### DISCUSSION THREAD

**Discussion by mushie4282** [OpenVoiceOS/ovos-buildroot#247] [Q&A] [answered]
For context, I am using March 2nd's raspOVOS image. Does anyone know if this indicates that my message bus is down? 
Any tips or guidance is welcome. I'm trying to better understand the messagebus before diving into skill creatation. 
![image](https://github.com/OpenVoiceOS/ovos-buildroot/assets/106356342/139e51a1-8c8c-4e7c-a444-ef6d8b32a6bc)

**Comment by mushie4282** [comment]
After viewing OVOS MessageBus's GitHub page. I should include my websocket config for better context. ━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━┓
┃ Configuration keys (Configuration: Joined, Section: websocket) ┃ Value   ┃
|━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
│ host                                                           │ 127.0.0.1            │
│ port                                                           │ 8181                 │
│ route                                                         │ /core                │
│ ssl                                                              │ False                │
│ shared_connection                                    │ True                 │
│ max_msg_size                                            │ 100                  │
└───────────────────────────────────────────────────

**Comment by JarbasAl** [comment]
it seems you are already running the bus and then trying to launch it again, but you cant run the bus twice so you get that error

it should already be running in the background automatically

**Reply by mushie4282** [reply]
oh okay. My intention was to watch what is being sent to the message bus in real time. So `ovos-messagebus` command is to relaunch the bus?

**Reply by JarbasAl** [reply, marked as answer]
yup, you should be able to just check the log files, nothing will launch if the bus isnt running so you would see a bunch of errors about connecting to the bus in all other logs (trying to reconnect in a loop)
